superx/routes/home.py

```diff
@@ -83,20 +114,30 @@ def add_item():
 
     return ''
 
+
 def remove_item():
-    '''
+    """
     removes item to cart using jquery to get the data and ajax so not to redirect
-    '''
+    """
     id_to_erase = request.form.get('id')
     if 'cart' not in session:
         return ''
 
     cart_list = session['cart']
-    for i in enumerate(cart_list):
+    for i in range(len(cart_list)):
         if cart_list[i]['id'] == id_to_erase:
             del cart_list[i]
             break
 
     session['cart'] = cart_list
 
     return ''
+
+
+def city():
+    """
+    get the city that the customer chose from the cities list
+    """
+    city = request.form.get("city")
+    session['city'] = city
```

Comment:
you can combine this together:
session['city'] = request.form.get("city"), 
but I'm not really sure what the purpose of this

Reply:
This is saving in session the city that the customer selected

Reply:
that part i understood. see the comment below, on why not get rid of this function and in the function
@app.route('/city', methods=['GET', 'POST'])
add in 
session['city'] = request.form.get("city")

Reply:
Done